This converts an hjson file that is being vendored in from Ibex so that it
matches a new format that dvsim expects. Instead of a list of values which
happen to have keys, we're now storing things as a key-value dictionary.

diff --git a/uvm/icache/dv/ibex_icache_sim_cfg.hjson b/uvm/icache/dv/ibex_icache_sim_cfg.hjson
index f1135870..20266605 100644
--- a/uvm/icache/dv/ibex_icache_sim_cfg.hjson
+++ b/uvm/icache/dv/ibex_icache_sim_cfg.hjson
@@ -29,12 +29,11 @@
 
   sim_tops: ["ibex_icache_fcov_bind"]
 
-  build_modes: [
-    {
-      name: default
+  build_modes: {
+    default: {
       en_build_modes: ["{tool}_memutil_dpi_scrambled_build_opts"]
     }
-  ]
+  }
 
   // Default iterations for all tests - each test entry can override this.
   reseed: 50
